### ISSUE TITLE
"Frontier server is down"

### ISSUE THREAD

**Issue by DjVayer99** [Marginal/EDMarketConnector#315]
I installed the app like 1 month ago. It never worked, always saying "Frontier server is down" and sometimes saying also things like "Can't connect to Inara/EDSM". I tried everything to make it work but nothing happened. I even added the tool to firewall exceptions and still nothing, any suggestions?

**Comment by Marginal** [comment]
There’s something about your firewall, proxy or antivirus settings that’s blocking the app from talking to the outside world.

**Comment by csanchezolt** [comment]
the same problem frontier server is down, 
path is ok: C:\Users\csanchez\Saved Games\Frontier Developments\Elite Dangerous

**Comment by connylindquist** [comment]
Just wanted to add I am seeing it to now. No changes has occured in my case and its been working flawlessly before yesterday.

**Comment by inorton** [comment]
You can see this if your ISP or router caches dns requests and if FDev have
changed the server ips or if your ISP DNS is broken.

I had this for quite a while on Virgin media.

As a check you could change your PC's DNS servers to ones like the Google
ones. 8.8.8.8 etc

**Comment by connylindquist** [comment]
Thank you for the tip but I can fairly easily determine its not a DNS issue. I am always behind a VPN tunnel with DNS leakprevention (using the vpn service DNS) so to test your theory I turned the VPN off and then tried to update again with my "real" ISP and their DNS and it was the same thing. Then I tested to to set google DNS (using my real ISP) as you suggested and it made no difference. I tried both google DNS 8.8.8.8 and 8.8.4.4. 

Is the login with the EDMC going to the same server as when I logon to play the actual game? (that works great by the way) Or do they have special logon servers as some games have? Also what ports does EDMC use? Thought I could test them and see if they got blocked for some reason after the windows updates.

**Comment by inorton** [comment]
Edmc logs into the frontier "companion API server" not exactly the same one as the game.

**Comment by DjVayer99** [comment]
I already tried to check what ports EDMC uses and there isn't a specific one so the problem can't be solved with that.

> Also what ports does EDMC use? Thought I could test them and see if they got blocked for some reason after the windows updates.

**Comment by Marginal** [comment]
@DjVayer99 The app tries to connect to some or all of:
companion.orerve.net and pts-companion.orerve.net on port 443
eddn.edcd.io on port 4430
edsm.net and inara.cz on port 443

Plus clicking on a ship link tries to open one of the following sites in your default browser:
http://www.edshipyard.com/
https://beta.coriolis.edcd.io/
https://coriolis.edcd.io/

**Comment by DjVayer99** [comment]
@Marginal apparently your tool hates me: I even opened these ports and i can't make it work yet...

**Comment by connylindquist** [comment]
Well, whatever the issue where its working again for me. I made no changes. Didnt have time to. :)

**Comment by DjVayer99** [comment]
Seems that the tool works fine on another PC connected to the same Network, but I still can't get it working on the "main" one. I've already tried to uninstall everything that could interfere with EDMC, including anti-virus, and disabled the firewall.

Any suggestions?

**Comment by Sparfell-git** [comment]
Hi

I haven't played ED for 2 weeks, played again yesterday but since then, frontier server is down for EDMC. I also use Trade Computer Extension which relies on EDMC, and found that others users have this same problem of server down for EDMC : https://forums.frontier.co.uk/showthread.php/223056-RELEASE-Trade-Computer-Extension-Mk-II/page386

Strangely, it did work very temporarily yesterday, so I am a bit lost regarding this and can only think of a problem on Frontier API server or EDMC server itself.

**Comment by MadHwK** [comment]
Greetings and stuff...

i do not precisely know what the technical difference between version 304 and the 303 is... but a roll back to the previous version fixed the "frontier server offline" issue for me.

de facto i have changed nothing relating my IPS, my Firewall or anything else what my connection relates...

the error must be hidden in the differences between the two versions.

**Comment by Lorithus** [comment]
Same problem here, only version 3.02 works fine for me.

**Comment by MartindForlon** [comment]
Same situation here, with latest EDMC version unable connect to FD servers (tried EDDI and there connection to FD server normally works and it updates market prices.

**Comment by mvolli** [comment]
reverted back to 3.03 too because of that issue.

**Comment by Quax456** [comment]
Roleback to 3.03 fixed the problem for me although....

Regards

**Comment by Marginal** [comment]
Please try [this](https://marginal.org.uk/EDMarketConnector_win_305b2.msi) test build and, once you see the error, zip up and attach here the file `%TMP%\EDMarketConnector.log`

**Comment by shukari** [comment]
Same problem, installed your version... in the log the time value seams wrong

[EDMarketConnector.log](https://github.com/Marginal/EDMarketConnector/files/1967096/EDMarketConnector.log)

**Comment by darth-crunchus** [comment]
Is this an issue with 3.2.0.0?  Because I, too, have been experiencing this problem, but it's only been since 3.3 rolled out officially, and the only reason I even noticed is because EDMC is part of my Elite 'EDC' as it were.

**Comment by Sharlikran** [comment]
I have 3.2.0.0 and I have not been playing for a long time. When I used this before it would ask me to provide my credentials and then I had to check my email for a code, enter the code, and that authorized this tool to access information.

Do I still have to do that?

One log I had:

[EDMarketConnector.log](https://github.com/Marginal/EDMarketConnector/files/2687758/EDMarketConnector.log)

First log file that gave me the impression that I had not done something correctly:

[EDMarketConnector - Org.log](https://github.com/Marginal/EDMarketConnector/files/2687764/EDMarketConnector.-.Org.log)
